ci/envoy_mobile_mirror.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -e
```

Comment:
heads up that this pattern already causes a race condition with the filter example (#22478 )